pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/creator/impl/stats/LongColumnPreIndexStatsCollector.java

```diff
@@ -57,10 +56,10 @@ public void collect(Object entry) {
     }
   }
 
-  void addressSorted(long entry) {
-    if (_isSorted) {
+  private void addressSorted(long entry) {
```

Comment:
I saw some of the `addressSorted` method use `private` but some don't. It'd be good to unify them.

Reply:
Done